lua/gk.lua

```diff
@@ -0,0 +1,28 @@
+local ffi = require("ffi")
+local gatekeeperc = require("gatekeeperc")
+
+local M = {}
+
+-- TODO Edit the GK configuration data.
+local gk_config = {
```

Comment:
Drop this variable and move assign the parameters directly to variable `conf` below.